Dockerfile

```diff
@@ -28,7 +28,12 @@ ADD Gemfile /tmp/Gemfile
 ADD Gemfile.lock /tmp/Gemfile.lock
 RUN bundle install --jobs 4
 
+
+
 ADD . /app
 WORKDIR /app
 
+RUN bundle exec whenever --update-crontab
+CMD service cron start
```

Comment:
Dockerfile is only run at image build time. It's not right to start the cron service here. It should be run at deployment time. We are in the process of automating that. I'll add a `.Jenkinsfile` today and we can add these lines there. Currently the deployment script is edited directly at our Jenkins instance. Please revert edits to this file.